packages/client/src/api/BaseAPI.ts

```diff
@@ -5,8 +5,10 @@ export default abstract class BaseAPI {
   protected endpoint: string;
   protected Transport: Transport = new Transport();
 
-  protected constructor(endpoint: string) {
-    this.endpoint = BaseAPI.API_URL + endpoint;
+  protected constructor(endpoint: string, apiUrl?: string) {
+    const baseUrl =
+      apiUrl !== undefined && apiUrl !== null ? apiUrl : BaseAPI.API_URL;
```

Comment:
сложно. давай сделаем проще 
```suggestion
const baseUrl = typeof apiUrl === 'string' ? apiUrl : BaseAPI.API_URL
```